docs/en/ingest-management/elastic-agent/configuration/providers/docker-provider.asciidoc

```diff
@@ -48,54 +32,56 @@ For example, the Docker provider provides the following inventory:
 [
     {
        "id": "1",
-       "mapping:": {"id": "1", "paths": {"log": "/var/log/containers/1.log"}},
+       "mapping:": { "container": {"id": "1", "name": "container-name", "image": {"name": "busybox:latest"}}},
        "processors": {"add_fields": {"container.name": "my-container"}}
     },
     {
         "id": "2",
-        "mapping": {"id": "2", "paths": {"log": "/var/log/containers/2.log"}},
+        "mapping:": { "container": {"id": "2", "name": "container-name2", "image": {"name": "busybox:latest"}}},
```

Comment:
looking on this example again - do you think it makes sense to add it to the public doc?
if I correctly understood, this example references to the [implementation](https://github.com/elastic/elastic-agent/blob/e7419ddc4e492e7e41acd20ec39f570937affe13/internal/pkg/composable/providers/docker/docker.go#L29-L33), since we have documentation in different repo - it is easy to forget changing it, so it can be outdated. Also it is not smth that customer can configure, so maybe just keep the `inputs:` and `generated policy` examples here?
@ChrsMark wdyt?

Reply:
Yeap, good point. I agree we can remove this since it's not user facing behavior but mostly internal implementation detail.

Reply:
done https://github.com/elastic/observability-docs/pull/2804/commits/0a56e4300240d3d03a3d6201afbf7781b388cb6c